.circleci/config.yml

```diff
@@ -64,28 +16,6 @@ jobs:
 workflows:
   version: 2
 
-  tests:
-    jobs:
-      - test-ruby-2.5: &test-filters
-          filters:
-            branches:
-              ignore:
-                - gh-pages
-                - gcloud-jsondoc
-                - gcloud-rdoc
-                - yard-gcloud
-      - test-ruby-2.4:
-          <<: *test-filters
-      - test-ruby-2.3:
-          <<: *test-filters
-
-  post-build:
```

Comment:
post-build is doing the docs build and the codecov.io push. Are those working on Kokoro?

Reply:
They are. I switched to commenting out instead of deleting so it's easier to fix if we discover issues.